### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by borod108**
Cover more main flows.

### CODE REVIEW

File: test/factories/host_factories.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    initiated_at nil
+    build_errors nil
+    discovery_rule_id nil
+
+    trait :with_interfaces do
+      interfaces { build_list :nic_kubevirt, 1 }
+    end
+  end
+
+  #factory :host_kubevirt_with_interfaces, parent: :host_kubevirt do |host_kubevirt|
```
Comment (masayag) [inline test/factories/host_factories.rb]: pls remove

**Comment by masayag** [comment]
Please make sure travis is happy and also `bundle exec rake` passes locally.

File: foreman_kubevirt.gemspec [hunk 2]
```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rdoc'
   s.add_development_dependency 'rubocop'
 
-  s.add_dependency('fog-kubevirt', '~>1.1')
+  s.add_dependency('fog-kubevirt', '~>1.2')
```
Comment (masayag) [inline foreman_kubevirt.gemspec]: this was merged in a separate PR, please rebase or remove it from this.

**Comment by borod108** [comment]
@masayag merge?

**Comment by pkliczewski** [comment]
:tada:

**Pull request merged**